### ISSUE TITLE
Indentation Screwed when reindenting

### ISSUE THREAD

**Issue by danielnehrig** [peitalin/vim-jsx-typescript#13]
```
const contentPanel = (array: JSX.Element[], dataType: string): JSX.Element | undefined => {
  if (array.length) {
    return (
      <div className='masonry-item col-md-6'>
        <div className='bgc-white p-20 bd'>
          <h6 className='c-grey-900'>{dataType}</h6>
          <ul>
              {array}
          </ul>
      </div>
    </div>
    )
  } else {
    return undefined
  }
}
```

reindent looks like this :
![Screenshot 2019-06-17 at 15 54 01](https://user-images.githubusercontent.com/4050749/59609693-26efc080-9118-11e9-8573-b8ce56ade3df.png)

Visual Select all and reindent with =
will screw up the ideation (that's caused by {array})

**Comment by mcabrams** [comment]
Same issue here, curious if anyone has found a solution.

**Comment by peitalin** [comment]
Should be fixed with last commit mentioned in #4 .
Not sure if this is a separate issue.
